### PR TITLE
Add dbgpt introducing. (#1221)

### PR DESCRIPTION
**Pull request by csunny**
DB-GPT is an experimental open-source project that uses localized GPT large models to interact with your data and environment. With this solution, you can be assured that there is no risk of data leakage, and your data is 100% private and secure.

This is the introducing of [DB-GPT](https://github.com/csunny/DB-GPT)

### CODE REVIEW

File: introducing-private-hub.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Introducing the Private Hub: A New Way to Build With Machine Learning"
-thumbnail: /blog/assets/92_introducing_private_hub/thumbnail.png
+thumbnail: ../assets/92_introducing_private_hub/thumbnail.png
```
Comment (julien-c) [inline introducing-private-hub.md:3]: wrong change

Comment (csunny) [reply introducing-private-hub.md]: already fix this.

**Review by osanseviero — COMMENTED** [review]
Hi there! Thanks for the PR!

Did you discuss this with a HF team member before opening this PR? The usual approach is to either discuss the topic and open an issue before opening the PR. I might not have visibility into existing discussions, though :)

**Comment by julien-c** [comment]
where did you publish the blogpost in the end @csunny?

**Pull request closed without merging**